org.widelands.widelands.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: org.widelands.widelands
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: widelands
+rename-icon: widelands
+rename-desktop-file: widelands.desktop
+rename-appdata-file: widelands.appdata.xml
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Play sounds
+  - --socket=pulseaudio
+  # Needs to talk to the network
+  - --share=network
+  - --persist=.widelands
+  # OpenGL access
+  - --device=dri
+build-options:
+  cflags: -O2 -g -Wno-deprecated-declarations -fstack-protector-strong -D_FORTIFY_SOURCE=2
```

Comment:
You can remove at least part of this. See `flatpak run --command=cat org.freedesktop.Sdk//18.08 /etc/flatpak-builder/defaults.json`

Reply:
Done.
https://github.com/scx/flathub/commit/4b76d59262f97c3072304f13dbe4a3e1fc2be742